AutomatedTesting/Gem/PythonTests/Physics/TestSuite_Main.py

```diff
@@ -3,92 +3,349 @@
 For complete copyright and license terms please see the LICENSE at the root of this distribution.
```

Comment:
This was moved TestSuite_Main_Optimized -> TestSuite_Main

Its pretty annoying its not able to tell in the change

Reply:
This is because ```TestSuite_Main.py``` was already in the repo, so it will just appear as changes to the file. Unless you do some extra steps / work (delete old file + commit, then do git mv to rename + commit).

Reply:
I actually did that if you check the commits, nevermind anyways..